python/mathsys/main/latex.py

```
@@ -48,30 +106,28 @@ def run(self, sheet: Level1) -> str:
-            case _:
-                self.latex.append("$$")
-                for statement in sheet.statements:
-                    match statement:
-                        case Declaration(): self.declaration(statement)
-                        case Node(): self.node(statement)
-                        case Equation(): self.equation(statement)
-                        case Comment(): self.comment(statement)
-                    self.latex.append(r"\\ ")
-                self.latex.pop()
-                self.latex.append("$$")
+            case 0: delimiter = ""
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)